compiler/codegen/OMRCodeGenPhaseFunctionTable.hpp

```diff
@@ -26,6 +26,7 @@
    TR::CodeGenPhase::performReserveCodeCachePhase,                                           //ReserveCodeCachePhase
    TR::CodeGenPhase::performLowerTreesPhase,                                                 //LowerTreesPhase
    TR::CodeGenPhase::performSetupForInstructionSelectionPhase,                               //SetupForInstructionSelectionPhase
+   TR::CodeGenPhase::performUncommonCallConstNodesPhase,                                     //UncommonCallConstNodesPhase
```

Comment:
`RemoveUnusedLocals` doesn't do anything to the trees or nodes, it just scans the list of autos and removes unreferenced symbols, so it's OK.